menas/ui/components/dataset/datasetDetail.controller.js

```diff
@@ -15,8 +15,10 @@
 sap.ui.define([
   "sap/ui/core/mvc/Controller",
   "sap/ui/core/Fragment",
-  "sap/m/MessageBox"
-], function (Controller, Fragment, MessageBox) {
+  "sap/m/MessageItem",
+  "sap/m/MessageBox",
+  "sap/m/MessagePopover"
```

Comment:
Is there a future reason to have these three here?

Reply:
No, that must have been a left over. Thanks for spotting it.